scipy/signal/windows/windows.py

```diff
@@ -1640,6 +1641,60 @@ def exponential(M, center=None, tau=1., sym=True):
     return _truncate(w, needs_trunc)
 
 
+def taylor(N, nbar=4, sll=-30):
```

Comment:
`sll` does not seem like a descriptive enough name. Can you look at the filter design functions to see if there is an existing name you could use? For example, in `kaiser` we use `ripple`:

http://scipy.github.io/devdocs/generated/scipy.signal.kaiserord.html#scipy.signal.kaiserord

It's not exactly applicable here, but maybe there is something we use already.

If there isn't an existing name that fits, something like `stop_rejection` would be more readable.

Reply:
The name has been changed to "level". In the reference it is called "SLL" for "side lobe level". Is "level" OK or perhaps "sllevel"? I think the "sl" portion is meaningful in this. Additionally, most other implementations of this Window use "SLL" (like in Matlab (https://www.mathworks.com/help/signal/ref/taylorwin.html) or Javascript (https://github.com/fasiha/taylorwin.js/)).

Reply:
Maybe `sidelobe=`?